.size-snapshot.json

```diff
@@ -1,137 +1,109 @@
 {
   "bundle.js": {
-    "bundled": 821205,
-    "minified": 752699,
-    "gzipped": 90131
+    "bundled": 815565,
```

Comment:
It looks like we'll want to update this again--let's wait until right before merging in so it's not being updated throughout PR fixes.